librz/core/canalysis.c

```diff
@@ -65,6 +65,7 @@ static bool find_string_at(RzCore *core, RzBinObject *bobj, ut64 pointer, char *
 	RzBin *bin = core->bin;
 	ut8 buffer[512] = { 0 };
 	bool ret = false;
+	bool big_endian = core->analysis->big_endian;
```

Comment:
since you have access to core, isn't better to call rz_config ?

Reply:
Not sure, taking it from RzAnalysis in `librz/core/canalysis.c` makes more sense, IMHO.